src/styles/index.ts

```diff
@@ -1181,6 +1181,9 @@ const styles = (theme: ThemeColors) =>
             overflow: 'hidden',
             borderBottomWidth: 2,
             borderColor: theme.border,
+        },
+
+        autoGrowMarkdwonTextInputPaddingBottom: {
```

Comment:
Rather than write a new style for this, why not just use `pb2` - it's a utility class that already exists.

Reply:
+1